[Docs] Clarify that _ccr/info omits parameters from the response when the follower index is paused. (#55961)
elastic/elasticsearch: docs/reference/ccr/apis/follow/get-follow-info.asciidoc

//Begin parameters
`parameters`::

> (object) An object that encapsulates {ccr} parameters. This object will be

Reviewer: Not a big deal, but I usually put the conditional part of the sentence first so folks can stop reading when it doesn't apply to them.
```suggestion
(object) An object that encapsulates {ccr} parameters. If the follower index's status is `paused`,
```

> omitted if the follower index's `status` is `paused`.

Reviewer: ...continued:
```suggestion
this object is omitted.
```

GET /follower_index/_ccr/info
--------------------------------------------------

> The API returns the following results if the follower index is `active`:

Reviewer: ```suggestion
If the follower index is `active`, the API returns the following results:
```

    ]
}
--------------------------------------------------

> The API returns the following results if the follower index is `paused`:

Reviewer: ```suggestion
 If the follower index is `paused`, the API returns the following results:
```

[source,console-result]
--------------------------------------------------
{
    "follower_indices" : [
        {
            "follower_index" : "follower_index",
            "remote_cluster" : "remote_cluster",
            "leader_index" : "leader_index",
            "status" : "paused"
        }
    ]
}

> --------------------------------------------------

Reviewer: This snippet is failing with "status: expected String [paused] but was String [active]".  I think we need to skip testing on this snippet or run a pause API and rerun the info API before this snippet.
